server/game/cards/events/01/risenfromthesea.js

```diff
@@ -14,6 +14,7 @@ class RisenFromTheSea extends DrawCard {
             handler: context => {
                 context.event.saveCard(context.target);
                 this.controller.attach(this.controller, this, context.target, 'play');
+                this.setCardType('attachment');
```

Comment:
Hmm. Where would that leave the 'Condition' trait it gains? Technically it's outside the quotes that specify the text.

I guess we can worry about it if there are ever any attachment blankers released. I get the feeling an until-leaves-play lasting effect would be a better approach, but may not be worth it right now.